CONTRIBUTING.md

```diff
@@ -5,3 +5,55 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommend code styling and best practices for all the Web Monitoring repositories. We also have some best practices specifically for our [web-monitoring-ui](https://github.com/edgi-govdata-archiving/web-monitoring-ui/blob/master/CONTRIBUTING.md) repo.
+
+## Code Style / Best Practices:
+
+### Naming:
+
+*No abbreviations.* Abbreviations tend to overlap a lot between domains, so they can be very ambiguous and confusing. This is especially true for non-native English speakers, even if they are fluent. Exceptions for things that are primarily referred to by abbreviation _in common language_, e.g. “URL,” “FAQ,” “ID,” _not_ “utils,” “idx,” etc. Single-letter names are ok only in contexts where their value is totally arbitrary, e.g. `function compare (a, b)`, or in the rare places where their use is normal, e.g. “x/y/z” in math functions referring to points.
+
+*If camel-casing, treat acronyms as a single word.* e.g. `parseHtml()`, not `parseHTML()`. This goes with the above—word boundaries can get easily confused when two acronyms run up against each other.
+
+### Alphabetize imports at the start of files:
+Keep it simple, just alphabetize at the top of the file.
+
+### Spacing in code:
+
+Overall, we recommend [Stroustrup](https://en.wikipedia.org/wiki/Indentation_style#Variant:_Stroustrup) spacing.
+'''if (foo) {
```

Comment:
These should be triple back-ticks, not quotes, so they mark this off as a code sample. They should also be on a separate line from the code and it would also be good to set the syntax as JS:

```
  ```js
  code
```

Reply:
Fixed.